generators/languages/templates/src/main/resources/i18n/_messages_zh_CN.properties

```diff
@@ -30,18 +30,18 @@ email.activation.text2=\u795D\u60A8\u4F7F\u7528\u6109\u5FEB,
 email.signature=<%= baseName %> \u56e2\u961f.
 
 # Creation email
-email.creation.text1=Your <%= baseName %> account has been created, please click on the URL below to access it:
+email.creation.text1=\u60a8\u7684 <%= baseName %> \u0020\u0020\u8d26\u6237\u5df2\u7ecf\u521b\u5efa\uff0c\u0020\u8bf7\u70b9\u51fb\u4ee5\u540e\u94fe\u63a5\u521b\u5efa\uff1a
```

Comment:
You can use the actual characters here as this looks very cryptic and is not user friendly